### PR TITLE
wordy: updated exercise generator to new test generator

### PR DESCRIPTION
**Pull request by eklatzer**
One of #2302

### CODE REVIEW

File: exercises/practice/wordy/cases_test.go [hunk 1]
```diff
@@ -1,159 +1,152 @@
...
+		expected:    0,
 	},
 	{
-		"reject prefix notation",
-		"What is plus 1 2?",
-		false,
-		0,
-	},
-	{
-		"reject using default 0 minus number",
```
Comment (andrerfcsantos) [inline exercises/practice/wordy/cases_test.go:154]: It seems this test is no longer present, but the `tests.toml` was not updated?

**Review by andrerfcsantos — CHANGES_REQUESTED** [review]

File: exercises/practice/wordy/cases_test.go (hunk 1 above)
Comment (eklatzer) [reply exercises/practice/wordy/cases_test.go:154]: Seems like it was added directly to `cases_test.go`: https://github.com/exercism/go/commit/e1a2b21f85a930e42bd683e8073e233fa6c16a2b

Comment (eklatzer) [reply exercises/practice/wordy/cases_test.go:154]: Should I add it to `wordy_test.go` and then mention it in #1892 as additional test?

Comment (andrerfcsantos) [reply exercises/practice/wordy/cases_test.go:154]: Let's not include this test then. I'll make a note to investigate this further later, but at first sight it seems that if this test is important, it should have been added directly to the problem specifications repo. It doesn't seem a test related to any go-specific change to the exercise.

**Pull request merged**